code/game/machinery/suit_storage_unit.dm

```diff
@@ -752,7 +752,7 @@
 
 	//Clear the access reqs, disable the safeties, and open up all paintjobs.
 	user << "<span class='danger'>You run the sequencer across the interface, corrupting the operating protocols.</span>"
-	departments = list("Engineering","Mining","Medical","Security","Atmos","HAZMAT","Construction","Biohazard","Crowd Control","Emergency Medical Response","^%###^%$", "Charring")
+	departments = list("Engineering","Mining","Medical","Security","Atmos","HAZMAT","Construction","Biohazard","Crowd Control","Emergency Medical Response","Exploration","Pilot","^%###^%$", "Charring") //VORESTATION EDIT

```

Comment:
The way this was originally done is poor. On emag_act it should += things into the departments list, not replace it entirely. You should revert this change (it will not be very likely to come up... how often are our cyclers emagged?) and make a PR to Polaris to just add the two additional departments as departments += list("whatever","whatever"), the symbols one and charring.